frontend/app/schemas/application-history-service-schemas.server.ts

```diff
@@ -142,6 +142,83 @@ const applicationSchema = z.object({
   ),
 });
 
+export interface applicationResponse {
```

Comment:
Convention is for TypeScript interfaces to start with a capital letter. In my opinion, we don't need the `Response` suffix.

```suggestion
export interface Application {
```

Reply:
Just noticed there is a type named `Application`. In my opinion, that existing type should be named `ApplicationResponse` (since it is API response structure).

Reply:
Fixed(?): 4dddc042c12d4426462cf9e8f6eb625103dbf417